yelp.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 1.9.2'
+  spec.required_ruby_version = '>= 2.0.0'
```

Comment:
Not sure I agree with this; I'd only bump this when we start using features only available in 2.0.0.

If this is the first step before using 2.0.0 features, then I agree.